src/screens/registerStudent/components/MiddleStudentRegister/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { FormHelperText, MenuItem } from '@mui/material';
+import { Box } from '@mui/system';
+import { TextField } from '../../../../components/textField';
+import { TMiddleStudentRegisterHook } from '../../hooks/types';
+import {
+  BackButton,
+  Container,
+  ContainerContinue,
+  ContinueButton,
+  SelectField,
+  StyledForm,
+  TypographyTextRegister,
+} from './styles';
+
+const MiddleStudentRegister = (register: TMiddleStudentRegisterHook) => {
+  const EnrollmentError = () => {
+    if (!!register.enrollmentError)
+      return <FormHelperText error>{register.enrollmentError}</FormHelperText>;
+
+    return <></>;
+  };
+
+  return (
+    <Container>
+      <TypographyTextRegister variant="h4">
+        Cadastro de Aluno
+      </TypographyTextRegister>
+      <TypographyTextRegister sx={{ mt: '12px' }} variant="body1">
+        Agora fale um pouco sobre você e nos informe seus dados acadêmicos.
+      </TypographyTextRegister>
+      <StyledForm>
+        <Box sx={{ width: '100%' }}>
+          <TextField
+            value={register.enrollment}
```

Comment:
Como a matrícula tem tamanho máximo, você podia já limitar esse campo a esse tamanho:
```suggestion
            value={register.enrollment}
            inputProps={{ maxLength: 8 }},
```
OBS.: Confirma se o tamanho máximo é 8 mesmo.

Reply:
É 8 mesmo. Adicionei esse detalhe aqui